### ISSUE TITLE
Unavailable API

### ISSUE THREAD

**Issue by strichter** [pybricks/pybricks-api#2]
Hi,

I was very excited seeing messaging support in the pybricks API. However, I got stopped in my tracks today when I actually tried to use it. I am currently building a 2-brick model with my son and we were planning to use BT messaging.

Where can I find a pybricks-micropython version that has the API documented here?

**Comment by laurensvalk** [comment]
We're working hard to get everything online in the week to come. Great to see you found this already :)

You can get the latest build of `ev3dev` which now comes with `pybricks-micropython` here:

https://oss.jfrog.org/list/oss-snapshot-local/org/ev3dev/brickstrap/

**Comment by laurensvalk** [comment]
It is still a beta version, though, so there may be a few things here and there that are not up to speed with the documentation yet.

But messaging is already working. We'd love to hear about your experiences when you try it.

**Comment by strichter** [comment]
I just found it and I am thrilled! Thanks for getting back to me so fast!

Be assured, I will be beta testing all week for you!

**Comment by laurensvalk** [comment]
Excellent, much appreciated! Feel free to share your findings at https://github.com/pybricks/support

Extra note while you are testing:

We're trying to make sure that any user script written with version 1.0, will still work in this new release, despite some API changes. So if you still have any old scripts ready to be tried out, we'd love to hear if we broke anything.

**Comment by strichter** [comment]
Unfortunately, I just started with pybricks and the messaging was my first experiment beyond some trivial motor testing.

When I built a printer/plotter wiht my other sone last year, we used ev3dev2. But given how much faster micropython is (I am joyfully surprised) and how much tighter the controls seem to be, I will switch over completely even with the other project.

**Comment by laurensvalk** [comment]
No problem, I'm sure there's still plenty we have to fix in the 2.0 version alone :)

Yes, Pybricks has its own set of motor control drivers, which was a major piece of this whole project, so I'm glad its being noticed, and feedback on this is very welcome too.

**Comment by strichter** [comment]
Ok, Bluetooth is working well. I wish there would be some high-level connectivity control, but I will play around with it.

For example, I would like to leave the server running and restart the client as I keep developing. Having to restart both on every testing cycle is annoying. I am sure I can come up with a solution, but  a child learning this stuff will not. I know it is not easy to do, but a simple `is_connected()` function would be nice, so when the client disconnects, the server knows to wait for a connection again.

On the other hand, maybe it would be best to hide the connection stuff altogether and only deal with mailboxes. Code behind the scenes can deal with everything connection related.

**Comment by laurensvalk** [comment]
Thanks for your feedback, glad to hear you got Bluetooth working.

I've copied your feedback to an internal issue about ongoing Bluetooth improvements. Feel free to create a dedicated issue about it if you like though.

As far as this issue is concerned, I've just populated the [downloads page](https://pybricks.com/downloads/) with some links. It's very minimal, but at least it can be found now. The docs already have a link to this download page too. 

So I think this issue is ready to be closed as the very first public issue. Thanks for reporting! :)
